scripts/addVersionFiles.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Use udclient to add files to a component version
+#
+# The following environment variables are REQUIRED to be set prior to invocation of the script:
+# JAVA_HOME
+# DS_AUTH_TOKEN or DS_USERNAME/DS_PASSWORD
+# DS_WEB_URL
+# FILES_CMD
+# FILES_COMPONENTNAME
+# FILES_VERSIONNAME
+# FILES_BASE
+#
+# The following environment variables can be OPTIONALLY set prior to invocation of the script:
+# FILES_OFFSET
+# FILES_INCLUDE
+# FILES_EXCLUDE
+# FILES_SAVEEXECUTEBITS
+
+#set -x
+base_cmd=""
+
+# Create the command to execute
+if [ -z $FILES_CMD ]
+then
+  echo "udclient command not specified.  Exiting."
+  exit 1
+else
+  base_cmd="\"$FILES_CMD\" addVersionFiles"
+fi
+
+# Specify component name
+if [ -z $FILES_COMPONENTNAME ]
+then
+  echo "Component name not specified.  Exiting."
+  exit 1
+else
+  base_cmd+=" -component \"$FILES_COMPONENTNAME\""
+fi
+
+# Specify component version name
+if [ -z "$FILES_VERSIONNAME" ]
+then
+  echo "Version name not specified.  Exiting."
+  exit 1
+else
+  base_cmd+=" -version \"$FILES_VERSIONNAME\""
+fi
+
+# Specify local path
+if [ -z $FILES_BASE ]
+then
+  echo "No base files specified. Exiting."
```

Comment:
This is taken care of in the action because addVersionFiles.sh is not called if not base directory  specified.  This check is overkill but I didn't think it hurt anything.

Reply:
Gotcha, if the addVersionFiles.sh script is invoke, we will need a base directory.